js/google_ima.js

```diff
@@ -1623,6 +1703,40 @@ require("../html5-common/js/utils/utils.js");
             }
 
             break;
+          case eventType.IMPRESSION:
+
+            this.currentImpressionTime = new Date().valueOf();
+            var loadTime = this.currentImpressionTime - this.adResponseTime;
+            var protocol = "VAST";
+            if (ad && ad.g && ad.g.vpaid === true)
+            {
+              protocol = "VPAID";
+            }
+            var type = "unknown";
+            if (ad && ad.isLinear())
+            {
+              if (ad.getContentType().lastIndexOf("video", 0 ) === 0 )
```

Comment:
Does this check check all instances of video content type?

Reply:
That is the idea. I couldn't find a list of all possible values getContentType() returns. Most values seem to be of the format : <media type>/<file format>.  Where <media type> is a general class like video or image, and file format is mp3, png, etc...

Reply:
Sounds good.